apps/sms/js/notify.js

```diff
@@ -56,16 +73,22 @@
 
   var Notify = {
     ringtone: function notification_ringtone() {
-      if (settings['audio.volume.notification'] &&
-          settings['notification.ringtone']) {
-        ringtone(settings['notification.ringtone']);
-      }
+      return getSettings(
+        ['audio.volume.notification', 'notification.ringtone']
```

Comment:
Sorry for the missleading, I just mean 'string', not the actual settings value. I agree that there is no need to cache the settings value here.